docs/cloud-native-security/index.rst

```diff
@@ -126,76 +122,48 @@ include: review of the customer’s runbook, summary of findings and
-these events. The per-event fee will apply to events (SIEM-correlated
-findings). For example, if the native security products generate five findings
-that are delivered to the SIEM and, after correlation, the SIEM generates two
-events (one for two related findings and one for three related findings), the
-customer will be charged for two events only.
-
-Cloud Native Security service fees do not include the cost of the native
-security products themselves. The cost of the native products as well as the
-costs of supporting infrastructure components that are required to integrate
-the native products with the Rackspace SIEM are included in the AWS
-infrastructure charges portion of the customer's Rackspace bill.
-
-Initial deployment and configuration will be charged as a one-time
-onboarding fee.
+Cloud Native Security service fees do not include the cost of the native security products themselves. The cost of the native products as well as the costs of the infrastructure supporting the integration with the Rackspace SIEM are included in the AWS infrastructure charges portion of the customer's Rackspace bill.
+
+Initial deployment and configuration will be charged as a one-time onboarding fee.
+
+Customers can choose between 2 billing models for Cloud Native Security – percentage of AWS infrastructure billing model or per-event billing model.
+
+Percentage of AWS infrastructure billing model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Customers that opt for this billing model will be charged a set percentage of their total AWS infrastructure costs on accounts that are in scope for Cloud Native Security. Charges will be based on usage of all AWS services and marketplace fees on accounts in scope for Cloud Native Security.
+
+In order to provide valuable service and to help with prioritization, a minimum severity level has been defined for each native AWS security product (source of findings). Rackspace will only ingest, and respond to, findings at or above the defined minimum severity level for each product. Findings with severities below this threshold will not be processed by Rackspace. The minimum severity levels for each product are defined as:
+
+* IAM Access Analyzer – **LOW**
+* Amazon GuardDuty – **MEDIUM**
+* AWS Security Hub (Security Standards) – **HIGH**
+
+Per-event billing model
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Customers that opt for this billing model will be charged a set fee for every security event Rackspace responds to. The monthly charge for the service is calculated as the product of the number of events multiplied by the cost per event, or the minimum monthly fee, whichever is greater.
+
+The per-event fee will apply to events (SIEM-correlated findings). For example, if the native security products generate five findings that are delivered to the SIEM and, after correlation, the SIEM generates two events (one for two related findings and one for three related findings), the customer will be charged for two events only.
```

Comment:
... per-event fee applies to events  ...
... customer is charged ...

Reply:
updated